lib/core.js

```diff
@@ -167,3 +181,18 @@ internals.pickEach = (obj, prop) => {
         .filter(([, value]) => value && typeof value[prop] !== 'undefined')
         .reduce((collect, [key, value]) => ({ ...collect, [key]: value[prop] }), {});
 };
+
+internals.mapEachLeafFunction = (obj, map) => {
+
+    return Object.entries(obj).reduce((collect, [key, value]) => {
+
+        return {
+            ...collect,
+            [key]: typeof value === 'function' ?
+                map(value) :
+                (value && typeof value === 'object') ?
+                    internals.mapEachLeafFunction(value, map) :
+                    value
```

Comment:
oh, huh. what cases does returning plain `value` cover? is that kinda a selector to access a constant?
less a question about this PR, more about redux practice generally

Reply:
I didn't design it for any specific use-case.  I wanted `select` to be identical to `selectors` except with any functions in there bound to `state`, and that lead to me just ignoring unfamiliar values.  I suppose the other option would be to strip them out for `select`, but it seems like it would make `app.select` and `app.selectors` less "reflective" of each other.

Reply:
oh, gotcha, that makes sense. thanks for explaining! 🙏